src/shared/domain/definition/case-field.model.ts

```diff
@@ -26,20 +26,33 @@ export class CaseField implements Orderable {
   @Type(() => WizardPageField)
   wizardProps?: WizardPageField;
 
+  private _value: any;
+  private _items: any = [];
 
-  getValue?(): string {
-    if (this.field_type.type === 'DynamicList') {
-      return this.value ? this.value.value.code : '';
+  @Expose()
+  get value(): any {
+    if (this.field_type && this.field_type.type === 'DynamicList') {
+      return this._value && this._value.value ? this._value.value.code : this._value;
     } else {
-      return this.value;
+      return this._value;
     }
   }
 
-  getItems?(): any {
-    if (this.field_type.type === 'DynamicList') {
-      return this.value ? this.value.list_items : [];
+  set value(newValue: any) {
+    this._value = newValue;
+  }
+
+  @Expose()
+  get items(): any {
+    if (this.field_type && this.field_type.type === 'DynamicList') {
+      return this._value.list_items ? this._value.list_items : this._items;
     } else {
       return this.field_type.fixed_list_items;
     }
   }
+
+  set items(newItems: any) {
```

Comment:
can you rename `newItems` to `items` please

Reply:
done